legend-engine-executionPlan-execution-store-relational/src/main/java/org/finos/legend/engine/plan/execution/stores/relational/result/SQLExecutionResult.java

```diff
@@ -80,6 +80,7 @@ public SQLExecutionResult(List<ExecutionActivity> activities, SQLExecutionNode S
         {
             this.connection = connection;
             this.statement = connection.createStatement();
+            this.statement.setFetchSize(100);
```

Comment:
This is setting fetchSize to a specific value for every DB connection we make - I don't think we want to do this